cpp/src/gandiva/precompiled/types.h

```diff
@@ -251,6 +251,7 @@ gdv_timestamp castTIMESTAMP_utf8(int64_t execution_context, const char* input,
 gdv_timestamp castTIMESTAMP_date64(gdv_date64);
 gdv_timestamp castTIMESTAMP_int64(gdv_int64);
 gdv_date64 castDATE_timestamp(gdv_timestamp);
+int32_t castTIME_timestamp(int64_t timestamp_in_millis);
```

Comment:
when you are referring to time and timestamp its more clear to use the aliases gdv_time & gdv_timestamp

Reply:
Changes applied